native_client/deepspeech.cc

```diff
@@ -121,15 +128,45 @@ Model::infer(float* aMfcc, int aNFrames, int aFrameLen)
   Tensor n_frames(DT_INT32, TensorShape({1}));
   n_frames.scalar<int>()() = aNFrames;
 
+  // "Reshape_3" = logits, before the built-in CTC decoder is called.
+  const char* output_node_name = mPriv->scorer == NULL ? "output_node" : "Reshape_3";
```

Comment:
Does that means that we will be either having to choose between C++ codepath CTC Decoder + KenLM *or* graph-based CTC decoding ? If so, it means no choice at all once we add `tfcompile` in the equation, because we cannot rely on in-graph ctc decoding.

Reply:
I'd see more something that plugs non-graph based CTC decoding but either with or without KenLM.

Reply:
So I'm trying to keep this PR focused on the changes needed on top of master to support using the new decoder, and leaving the always decode in C++ stuff for your PR, since your needs could change anyway between now and landing.

Reply:
Good then :) 👍